docs/python/debugging.md

```diff
@@ -37,23 +37,27 @@ Standard configuration for `launch.json`:
     "name": "Python: Current File",
     "type": "python",
     "request": "launch",
-    "stopOnEntry": true,
-    "pythonPath":"${config:python.pythonPath}",
     "program": "${file}",
-    "cwd": "${workspaceFolder}",
-    "env": {},
-    "envFile": "${workspaceFolder}/.env",
-    "debugOptions": [
-        "RedirectOutput"
-    ]
 }
 ```
 
 Custom configurations for various settings are described in the following sections.
 
-### `pythonPath`
+### `name`
+
+Provides the name for the debug configuration that appears in the VS Code drop-down list.
+
+### `type`
+
+Identifies the type of debugger to use; leave this set to `python` for Python code.
 
-Points to the Python interpreter to be used for debugging purposes. The standard configuration uses the interpreter identified in the `python.pythonPath` setting by referring to `${config:python.pythonPath}`. To use a different interpreter, specify its path instead.
+### `request`
+
+Specifies the mode in which to start debugging:
+
+- `launch`: start the debugger on the file specified in `program`
+- `attach`: attach the debugger to an already running process. See [Remote debugging](#_remote-debugging) for an example.
+- `shell`: launches a separate program as specifies in `args`. See [Goodle App Engine debugging](#_google-app-engine-debugging) for an example.
```

Comment:
Let me get back to you on this.

Reply:
Typo: Goodle App Engine

Reply:
My bad...it should be under "type" as that's where it's used with the Google App Engine.